policy/modules.conf

```diff
@@ -3078,3 +3078,10 @@ fedoratp = module
 # stalld
 #
 stalld = module
+
+# Layer: contrib
+# Module: wireguard
+#
+# wireguard
+#
+wireguard = module
```

Comment:
I can't directly see why, but github requests rebasing.

Reply:
Should be fixed